main.py

```diff
@@ -1,7 +1,7 @@
-print(f'Hello {input("Enter your name: ")}!')
-age = int(input("Enter your age: "))
+name = input("Enter your name: \n")
+age = int(input("Enter your age: \n"))
 
-if age > 17:
-    print("You are allowed to take a drivers licence!")
+if age >= 18:
```

Comment:
We want to use 16 instead
```suggestion
if age >= 16:
```